README.md

```diff
@@ -83,7 +83,32 @@ data is sent to SDES for quarantining. It will be easier to understand this by r
 
 The embedded binary data is the value of the element named `includedBinaryObject`. This Base 64 encoded string is sent to SDES, accompanied
 by a metadata header, and SDES returns a UUID in its response.
-The values if the `includedBinaryObject` is replaced with the SDES UUID and the message is then forwarded to the configured 
+The value of the `includedBinaryObject` is replaced with the SDES UUID and the message is then forwarded to the configured 
 recipient service. Note that for ICS2, the UUID is itself Base 64 encoded before being inserted in the message. This is
 probably not necessary as a UUID wouldn't contain any characters that could violate an XML schema, but is the behaviour
 that is expected by the consumer of ICS2 messages `import-control-inbound-soap`.
+
+### Running on its own
+The service can be started locally using sbt. If you wanted to start it on the same port as does Service Manager (6707) you would
+excute ```sbt "run 6707"```
```

Comment:
Could we add config for this in build.sbt instead? Similar to https://github.com/hmrc/api-platform-outbound-soap/blob/main/build.sbt#L19